### ISSUE TITLE
Redesign the rule description page

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2059]
https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md

Should have multiple tables, separated by the following headings:

- WCAG 2.0 Level A & AA rules
- WCAG 2.1 Level A & AA rules
- Best practices rules
- Experimental rules
- Deprecated rules

Can take out the "enabled by default" column. Add a short intro after each heading indicating what is included, and if those rules are enabled by default. Combine the Failures, Needs Review columns into a single "Issue type" column, which can be "failure", "needs review" or "failure or needs review".

Content should be sorted alphabetically by rule ID within each category. Rule IDs should have links to DQU help pages. Add `?application=RuleDescription` to the end of each URL.

**Comment by jbottigliero** [comment]
Hey @WilcoFiers – I'm actually going through this information now to get a better idea of how much of the WCAG the standards are covered by default. Is this something actively being worked on? If not, I could try to put together a proposal...

**Comment by straker** [comment]
I'm currently working on it. Was there something you'd like changed that would help you understand it better?

**Comment by jbottigliero** [comment]
If the `Tags` column is carried over, I think the changes in #1309 would be an improvement (even if there needs to be some updates to upstream tooling before that feature is fully-realized)... but I think the structure proposed here would be much clearer overall. 

The information I'm personally trying to gather is a better understanding of what success criterion from the recommendations will _not_ be caught when using `axe-core`. If I could ask for _anything_ it'd be a [node.green](https://node.green/) or [`kangax/compat-table`](https://kangax.github.io/compat-table/es6/) style table with the mapping of _all_  WCAG success criterion (by version) to the status of support in `axe-core` – it doesn't seem like _this_ documentation is the place for that to exist but would certainly be helpful. 😅

**Comment by straker** [comment]
Interesting idea. The only problem we have with that is that even though we have rules that cover certain success criterion,  no WCAG criterion can be fully automated. So a list such as you've requested might present itself as axe-core covers all of the criterion when in can only cover a portion of it

**Comment by jbottigliero** [comment]
Totally agree that it would need to make clear that full coverage isn't possible via automation (even for some of the guidelines that _are_ present), but I think having a better understanding of the intersection of guidelines and `axe-core`'s rules might be helpful. Either way, probably a bit of scope-creep for this issue in particular...

**Comment by straker** [comment]
Testing: open the rule-description doc in github, ensure it meets the requirements outlined in the ticket. 
Docs: none

**Comment by padmavemulapati** [comment]
Went through rule-description document, observed the WCAG standard(s) are displaying in the tables of heading with seperate categories each, for all 
WCAG 2.0 Level A & AA rules
WCAG 2.1 Level A & AA rules
Best practices rules
Experimental rules
Deprecated rules.
